src/main/java/com/healthapp/project/Entity/Medication.java

```diff
@@ -0,0 +1,10 @@
+package com.healthapp.project.Entity;
+
+import io.norberg.automatter.AutoMatter;
+
+@AutoMatter
+public interface Medication {
+    int id();
+    String medication_name();
+    String strength();
+}
```

Comment:
So with a few lines, there's so many convenient things built in here. It also handles JSON mapping for me so i dont have to worry about that as well. I'll probably circle back and do the same to the User entity once i handle the medication api